src/app/pages/fund/components/zero-percent-fees-badge.tsx

```diff
@@ -20,7 +20,7 @@ export function ZeroPercentFeesBadge() {
       spacing="extra-tight"
     >
       <FiStar color="#F59300" size="12px" strokeWidth="2.5px" />
-      <Caption color="#FFA953" fontWeight={500} variant="c2">
+      <Caption color="#FFA953" fontWeight={500}>
```

Comment:
@fabric-8 @mica000 : Can you please help with this too? 

What new colour should we use for `#F59300` and `#FFA953` 

It's shown in this `0% fees` here:
![Screenshot 2023-11-09 at 09 54 26](https://github.com/leather-wallet/extension/assets/2938440/5451d3f1-eded-4982-93f7-2d9ef9ac2624)

Reply:
Michelly gave the colours for this here: https://www.figma.com/file/2MLHeIeL6XPVi3Tc2DfFCr/%E2%9D%96-Leather-%E2%80%93-Design-System?type=design&node-id=1608-66&mode=design&t=2wIcBIcHNQLISVon-0 

I'll update soon

Reply:
These colors are already available as `warning.label` and `success.label`. I will address this since was in my original PR.

Reply:
OK great, thank you!